libs/net/css/Extensions/HttpResponseMessageExtensions.cs

```diff
@@ -31,7 +31,7 @@ public static async Task<T> HandleResponseAsync<T>(this HttpResponseMessage resp
                         using var responseStream = await response.Content.ReadAsStreamAsync();
                         var result = await responseStream.DeserializeAsync<T>();
                         if (result != null) return result;
-                        else if (Nullable.GetUnderlyingType(typeof(T)) != null)
+                        else if (typeof(T).IsNullable())
```

Comment:
Handle nullable values.  This still isn't 100% correct, but I can't find an example to determine if `?` is being used in the function type...